python/dlr/tf_model.py

```diff
@@ -0,0 +1,143 @@
+import tensorflow as tf
+from tensorflow.python.tools import saved_model_utils
+
+from .api import IDLRModel
+from .neologger import create_logger
+from .metadata import VERSION
+
+class TFModelImpl(IDLRModel):
+    """
+    TFModelImpl is a wrapper on top of tensorflow which implements DLRModel API
+    Parameters
+    ----------
+    model_path : str
+        Full path to the directory containing the saved model
```

Comment:
It sounds like a parent dir of the saved model. Lets says "Full path to the saved model directory"

Reply:
done